docs/getting_started/sample-minimal-playbook_2_node_sap_hana_cluster.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
it is not clear what are prerequisites to run this playbook. 

HANA bits on both hosts? HANA installed? both hosts are registered? Both hosts are RHEL for SAP soltuions?